lib/node_modules/@stdlib/lapack/base/dpoequ/README.md

```diff
@@ -0,0 +1,265 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dpoequ
+
+> Compute row and column scalings intended to equilibrate a symmetric positive definite matrix `A`.
+
+<section class = "usage">
+
+## Usage
+
+```javascript
+var dpoequ = require( '@stdlib/lapack/base/dpoequ' );
+```
+
+#### dpoequ( order, N, A, LDA, S, scond, amax )
+
+Computes row and column scalings intended to equilibrate a symmetric positive definite matrix `A`.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var A = new Float64Array( [ 1.0, 2.0, 2.0, 4.0 ] );
+var S = new Float64Array( 2 );
+var scond = new Float64Array( 1 );
+var amax = new Float64Array( 1 );
+
+var out = dpoequ( 'row-major', 2, A, 2, S, scond, amax );
+// returns 0
+```
+
+The function has the following parameters:
+
+-   **order**: storage layout.
+-   **N**: order of matrix.
+-   **A**: input symmetric positive definite [`Float64Array`][mdn-float64array].
+-   **LDA**: stride of the first dimension of `A` (a.k.a., leading dimension of the matrix `A`).
+-   **S**: [`Float64Array`][mdn-float64array] containing scale factors.
+-   **scond**: [`Float64Array`][mdn-float64array] containing the ratio of the smallest to the largest scale factors.
+-   **amax**: [`Float64Array`][mdn-float64array] containing the absolute value of the largest matrix element.
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+// Initial arrays...
+var A0 = new Float64Array( [ 0.0, 1.0, 2.0, 2.0, 4.0 ] );
+var S0 = new Float64Array( [ 0.0, 0.0, 0.0 ] );
+var scond0 = new Float64Array( [ 0.0, 0.0 ] );
+var amax0 = new Float64Array( [ 0.0, 0.0 ] );
+
+// Create offset views...
+var A1 = new Float64Array( A0.buffer, A0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+var S1 = new Float64Array( S0.buffer, S0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+var scond1 = new Float64Array( scond0.buffer, scond0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+var amax1 = new Float64Array( amax0.buffer, amax0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+
+var out = dpoequ( 'row-major', 2, A1, 2, S1, scond1, amax1 );
+// returns 0
+```
+
+#### dpoequ.ndarray( N, A, sa1, sa2, oa, S, ss, os, sc, osc, am, oam )
+
+Computes row and column scalings intended to equilibrate a symmetric positive definite matrix `A` using alternative semantic indexing.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var A = new Float64Array( [ 1.0, 2.0, 2.0, 4.0 ] );
+var S = new Float64Array( 2 );
+var scond = new Float64Array( 1 );
+var amax = new Float64Array( 1 );
+
+var out = dpoequ.ndarray( 2, A, 2, 1, 0, S, 1, 0, scond, 0, amax, 0 );
+// returns 0
+```
+
+The function has the following parameters:
+
+-   **N**: order of matrix.
```

Comment:
Description is wrong.

Reply:
Or I suppose by "order", you mean the number of elements along each dimension of a square matrix `A`.